doc/classes/PhysicsServer2D.xml

```diff
@@ -369,7 +369,8 @@
 		<method name="body_create">
 			<return type="RID" />
 			<description>
-				Creates a 2D body object in the physics server, and returns the [RID] that identifies it. Use [method body_add_shape] to add shapes to it, use [method body_set_state] to set its transform, and use [method body_set_space] to add the body to a space.
+				Creates a 2D body object in the physics server, and returns the [RID] that identifies it. The default settings for the created area include a collision layer and mask set to [code]1[/code], and [enum BodyMode] set to [code]BODY_MODE_RIGID[/code].
```

Comment:
```suggestion
				Creates a 2D body object in the physics server, and returns the [RID] that identifies it. The default settings for the created area include a collision layer and mask set to [code]1[/code], and body mode set to [constant BODY_MODE_RIGID].
```

Same for 3D.

*Note:* Please make modifications by _amending_ the commit, and *force pushing* the changes, so that it stays as a single commit (see [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#modifying-a-pull-request)).